### DISCUSSION TITLE
No border work effect is not good？

### DISCUSSION THREAD

**Discussion by lixingjuan** [webcrumbs-community/webcrumbs#241] [Q&A]
![企业微信截图_c0bced3e-b7cd-4620-9b1f-3a40239559b9](https://github.com/webcrumbs-community/webcrumbs/assets/42627664/b15bace8-95ad-4290-a512-ad98ac1b86c5)

it can't generate a preview image, effect ⬇️
![image](https://github.com/webcrumbs-community/webcrumbs/assets/42627664/f06bf05f-dad9-479a-ad11-bb64f2f5adbc)

**Comment by pachicodes** [comment]
Hi there @lixingjuan 
Sometimes when the image is too big it can't create a preview, but it still creates the code.
At the moment it works better for components, rather than a full page, but we will improve it in the future.

Thanks for the feedback!
